adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -282,18 +281,35 @@ case class ParquetUnboundFeatureRDD private[rdd] (
     sqlContext.read.parquet(parquetFilename).as[FeatureProduct]
   }
 
+  /**
+   * Replaces the sequence dictionary attached to a ParquetUnboundFeatureRDD.
+   *
+   * @param newSequences The new sequence dictionary to attach.
+   * @return Returns a new ParquetUnboundFeatureRDD with the sequences replaced.
+   */
   def replaceSequences(newSequences: SequenceDictionary): FeatureRDD = {
     copy(sequences = newSequences)
   }
 
+  /**
+   * Replaces the sample metadata attached to the ParquetUnboundFeatureRDD.
+   *
+   * @param newSamples The new sample metadata to attach.
+   * @return A ParquetUnboundFeatureRDD with new sample metadata.
+   */
+  def replaceSamples(newSamples: Iterable[Sample]): FeatureRDD = {
```

Comment:
This method is defined with scaladoc in `MultisampleGenomicDataset`, could we use `override def` instead?